lib/gridstore/src/tracker.rs

```diff
@@ -149,17 +141,20 @@ impl PointerUpdates {
             return true;
         }
 
-        let Self { set, unset } = persisted;
+        let Self {
+            current: previous_current,
+            to_free: freed,
+        } = persisted;
```

Comment:
I wanted to nit about `to_free` in context of things that are already persisted. But I like the way you solved that :pray: